test/DynamoCoreTests/AuthenticationManagerTests.cs

```diff
@@ -74,15 +74,6 @@ public void ToggleLoginState()
             Assert.True(loginCalled);
         }
 
-        [Test]
-        public void CanToggleLoginState()
-        {
-            var authProviderMock = new Mock<IAuthProvider>();
-            var authManager = new AuthenticationManager(authProviderMock.Object);
-
-            Assert.IsTrue(authManager.CanToggleLoginState());
-        }
```

Comment:
@reddyashish Does this test fail only in service mode or? Would you share the reason we remove it?

Reply:
@QilongTang This change was made by Filipe here https://github.com/DynamoDS/Dynamo/pull/13657, to remove legacy code. We need to sync this branch with master.

Reply:
@reddyashish Thanks!